contracts/LiquidPledging.sol

```diff
@@ -195,23 +183,18 @@ contract LiquidPledging is LiquidPledgingBase {
                 return;
             }
         }
-        assert(false);  // It is not the owner nor any delegate.
+        assert(false);  // When the sender is not an owner or a delegate 
     }
 
-    /// @notice This method is used to withdraw value from the system.
-    ///  This can be used by the givers withdraw any un-commited donations.
+    /// @notice Authorizes a payment be made from the `vault` can be used by the
```

Comment:
authorized or withdrawn is dependent on the vault. If `autopay = true` in the vault, then the funds are withdrawn, otherwise, they must be confirmed

Reply:
`confirmPayment()` is always called tho, so it makes sense to say this authorizes the payment and the confirm function executes it